src/ui/components/results/verticalresultscomponent.js

```diff
@@ -128,20 +129,28 @@ class VerticalResultsConfig {
        * The aria-label given to the applied filters bar. Defaults to 'Filters applied to this search:'.
        * @type {string}
        **/
-      labelText: defaultConfigOption(config, ['appliedFilters.labelText'], 'Filters applied to this search:'),
+      labelText: defaultConfigOption(config, ['appliedFilters.labelText'], TranslationFlagger.flag({
```

Comment:
LL. If all args can't fit on the same line as the function name, see if they can all fit on a separate newline after the function name. If that won't work either, each arg should begin on its own newline.

Reply:
Updating! Same question as in this comment about consistent LL in this repo https://github.com/yext/answers/pull/1055/files#r482162852

Reply:
I try and stick to 90 characters or less. We have not actual standard in this repo (which is a problem). Oliver and I have been discussing some linting we could add to really codify this.